p4-16/spec/P4-16-spec.mdk

```diff
@@ -2025,7 +2027,7 @@ types. For additional examples of literals see Section
 | `8s10`  | Type is `int<8>`, value is 10 |
 | `2s3`   | Type is `int<2>`, value is -1 (last 2 bits), overflow warning |
 | `1w10`  | Type is `bit<1>`, value is 0 (last bit), overflow warning |
-| `1s10`  | Error: 1-bit signed type is illegal |
+| `1s10`  | Type is `int<1>`, value is -1, overflow warning |
```

Comment:
This seems very odd to me.  If the example were

`1s1` Type is `int<1>`, value is -1

that would make sense to me.

Why should it be allowed if two bits are given after the 's'?

Reply:
I will change this, since it is wrong. But that's more than two bits, it's ten in base ten.